submissions/7SergSmith7/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>HTML-CSS-Popup</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
```

Comment:
Why you add `.container` and `<header>` without adding styles to him?

Reply:
For semantic markup, I think )